rts8891.c:5392:11: note: include the header <math.h> or explicitly provide a declaration for 'fabs'

Index: backend/rts8891.c
--- backend/rts8891.c.orig
+++ backend/rts8891.c
@@ -76,6 +76,7 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <limits.h>
+#include <math.h>
 #include <signal.h>
 #include <stdio.h>
 #include <stdlib.h>
